### PR TITLE
[MTSA][Refactor Reachability][Part 2/2] Replace old Reachability implementation.

### PR DESCRIPTION
**Pull request by danielVebman**
## Overview

Finished refactoring Reachability.

## Changes Made

Replaced all old references to Reachability with the central manager.

## Test Coverage

Ran the app to make sure it still responds to reachability updates.

## Related PRs or Issues

PR #348. Issue #340.

### CODE REVIEW

**Review by Omarrasheed — APPROVED** [review]
Looks great!

File: TCAT/Controllers/CustomNavigationController.swift [hunk 1]
```diff
@@ -139,25 +105,9 @@ class CustomNavigationController: UINavigationController, UINavigationController
...
-    @objc func reachabilityChanged(notif: Notification) {
-        if let reachability = notif.object as? Reachability {
-            switch reachability.connection {
-            case .wifi, .cellular:
-                banner.dismiss()
-            case .none:
-                banner.show(queuePosition: .front, on: self)
-                banner.autoDismiss = false
-                banner.isUserInteractionEnabled = false
-            }
```
Comment (Omarrasheed) [inline TCAT/Controllers/CustomNavigationController.swift:151]: Are we not showing the banner anymore? I don't see that this code was moved anywhere, just deleted

**Review by Omarrasheed — CHANGES_REQUESTED** [review]

File: TCAT/Controllers/CustomNavigationController.swift (hunk 1 above)
Comment (danielVebman) [reply TCAT/Controllers/CustomNavigationController.swift:151]: It's back now 😬

**Review by Omarrasheed — APPROVED** [review]
Looks good again! J quickly run `swiftlint autocorrect` before merging

**Comment by danielVebman** [comment]
> Looks good again! J quickly run `swiftlint autocorrect` before merging

@Omarrasheed Is there a typo/formatting issue somewhere or just just because?

**Comment by Omarrasheed** [comment]
> > Looks good again! J quickly run `swiftlint autocorrect` before merging
> 
> @Omarrasheed Is there a typo/formatting issue somewhere or just just because?

Seems like ur last commit added a lotta spaces on newlines

**Review by kevinchan159 — APPROVED** [review]
looks good!

**Comment by danielVebman** [comment]
> > > Looks good again! J quickly run `swiftlint autocorrect` before merging
> > 
> > 
> > @Omarrasheed Is there a typo/formatting issue somewhere or just just because?
> 
> Seems like ur last commit added a lotta spaces on newlines

Better?

**Comment by Omarrasheed** [comment]
@danielVebman Perfect!

**Pull request merged**